### ISSUE TITLE
Possible to show *'s when text entered in terminal

### ISSUE THREAD

**Issue by aharpervc** [conradkleinespel/rpassword#85]
Hello! When using this library to prompt for a password, eg like `rpassword::prompt_password_stdout("Password: ")`, there's nothing shown on the terminal when text is entered.

Is it possible to have something like `***`'s be shown instead? Then it will be more clear to the user they have entered some text.

Thanks!

**Comment by conradkleinespel** [comment]
Hi @aharpervc, I appreciate your question. It is not currently possible to do that. I'm not inclined to add code to support this because:
- the standard way on Unix to ask for passwords is not to show anything, which you can see when running commands like `sudo su` or `ssh-keygen`;
- this would require time to develop and test properly, which at the moment I don't have. 

Feel free to fork the repo and implement this if you really need it though! If you do, please share your fork here, maybe at some point we can reconcile both repositories.

**Comment by aharpervc** [comment]
Okay thanks for the response... I'll think about it

**Comment by chipsenkbeil** [comment]
Old issue, but wanted to comment that this would be a nice feature (eventually) to include.

I learned that there was a way to make this happen for `sudo` where you can add `pwfeedback` to `/etc/sudoers` to enable stars/asterisks as feedback when adding a password.

Also learned that `sudo-rs` changed the default upstream to show stars, and now Ubuntu has inherited this: https://www.phoronix.com/news/sudo-rs-password-feedback

1. Wondering if a contribution (eventually) would be able to support reading `/etc/sudoers` to see if `pwfeedback` was enabled and if so follow suit.
2. Having an option for forcing stars or something else would also be handy

Maybe something like:

```
pub enum PasswordVisibility {
    Hide, // Default that shows nothing
    Stars, // Shows stars for all input
    PartialStars(N), // Shows for N characters before stars
}

pub fn prompt_password_visible(prompt: impl ToString, visibility: PasswordVisibility) -> std::io::Result<String> {
    // ...
}
```

**Comment by conradkleinespel** [comment]
Re-opening this since @chipsenkbeil was able to submit a PR and it's good. Thanks, @chipsenkbeil! We can close this issue again once the PR is merged. In the meantime, @aharpervc if you're still around and have feedback on the implementation, let us know.

**Comment by aharpervc** [comment]
I switched my app over to the fork and used prompt_password_with_feedback w/ rpassword::PasswordFeedback::Mask('*'), and it works fine by me

**Comment by conradkleinespel** [comment]
Thanks, @aharpervc, for testing, good to know! The API has changed a bit since your last comment, you can find an example here: https://github.com/conradkleinespel/rpassword/blob/810260a6da3e3f5dcb7c8102ebf4aee4fd3d192d/examples/prompt_with_config.rs#L14-L17

Thanks, @chipsenkbeil, for your contribution!

There is a tiny bit of work left before I publish to crates.io. The PR from @chipsenkbeil made me realize the pre-existing code didn't handle arrow keys correctly. So I'm leaving this issue open for now. @chipsenkbeil let me know if you're taking a look, otherwise, I will have a look within the next few days.

**Comment by chipsenkbeil** [comment]
Won't be able to contribute again for a couple of weeks, so go for it :)

**Comment by conradkleinespel** [comment]
Sounds good, thanks for letting me know!

**Comment by conradkleinespel** [comment]
Hi there, I'm back with some news and an ask.

In the last few days, I've made some more changes. While everything *should* be backwards compatible, Murphy's law tells me a bit more testing is a good idea.

Before I publish this new version as `7.5.0` to https://crates.io, are able to test this new code from the `main` branch and give me feedback ?  You'll find new documentation in the upcoming README.md, [viewable here in another branch](https://github.com/conradkleinespel/rpassword/tree/v750) or within the `examples/` directory.

**Comment by conradkleinespel** [comment]
I've just published v7.5.0 👍  thanks again you guys for your contributions
